ngraph_onnx/onnx_importer/backend.py

```diff
@@ -79,8 +79,12 @@ def supports_ngraph_device(cls, ngraph_device_name):  # type: (str) -> bool
         if ngraph_device_name not in cls._get_supported_devices():
             try:
                 ng.runtime(backend_name=ngraph_device_name)
-            except RuntimeError:
-                return False
+            except RuntimeError as e:
+                expected_err_msg = 'Backend \'' + ngraph_device_name + '\' not found in registered backends'
```

Comment:
This is a very stringent condition. If the Exception text is changed by even a single character, this will fail.
Let's relax it to something like `if 'not found' in str(e):`
You can add the reason for this in a comment.